src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -94,6 +110,7 @@ const generateWorkspaceSavedObjectsClientWrapper = () => {
     requestMock,
   };
 };
+const isDashboardAdmin = true;
```

Comment:
A little bit confusing as we are using the same name `isDashboardAdmin` in the parameters of `generateWorkspaceSavedObjectsClientWrapper` and here, can we change any one of them to another name?

Reply:
Yes, I had done it.